modules/administration-guide/partials/proc_collecting-dev-workspace-operator-metrics-with-prometheus.adoc

```diff
@@ -1,114 +1,83 @@
 [id="proc_collecting-dev-workspace-operator-metrics-with-prometheus"]
-= Collecting {devworkspace} Operator metrics with Prometheus
+= Collecting {devworkspace} Operator metrics
 
 [role="_abstract"]
-To use Prometheus to collect, store, and query metrics about the {devworkspace} Operator:
+To use the in-cluster Prometheus instance to collect, store, and query metrics about the {devworkspace} Operator:
 
 .Prerequisites
 
-* The `devworkspace-controller-metrics` Service is exposing metrics on port `8443`. This is preconfigured by default.
+* An active `{orch-cli}` session with administrative permissions to the destination {orch-name} cluster. See {orch-cli-link}.
 
-* The `devworkspace-webhookserver` Service is exposing metrics on port `9443`. This is preconfigured by default.
+* An instance of {prod-short} running in {orch-name}.
```

Comment:
Since this section is only applicable to OpenShift and not vanilla Kubernetes, adding this suggestion to hardcode the attribute/variable values because the upstream docs show Kubernetes:
```suggestion
* Your organization's instance of {prod-short} is installed and running in Red Hat OpenShift.

* An active `oc` session with administrative permissions to the destination OpenShift cluster. See link:https://docs.openshift.com/container-platform/{ocp4-ver}/cli_reference/openshift_cli/getting-started-cli.html[Getting started with the CLI].
```

Reply:
Sounds good, I wasn't able to apply this suggestion from the GitHub UI, so I've pushed a new commit